src/Solr/Solarium/Adapter.php

```diff
@@ -615,6 +615,18 @@ protected function applyParametersOnQuery(
                 $query->setSorts($sortings);
             }
 
+            $queryParser = $parameters->getQueryParser();
```

Comment:
Ich verstehe den Impuls beliebige QueryParser, beliebige Strings, zu erlauben, aber macht er Sinn? Wir bringen damit etwas Solr-spezifisches in die OPUS 4 Seite der Implementation. Die Adapter-Klasse ist für Solr implementiert, aber der QueryParser-String kommt ja von außen, über die Query-Klasse rein.